webdriver/tests/interaction/element_click.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click
+
+def test_no_browsing_context(session, create_window):
+    # 14.1 step 1
+    session.url = inline("<p>This is not an editable paragraph.")
+    element = session.find.css("p", all=False)
+
+    session.window_handle = create_window()
+    session.close()
+
+    response = click(session, element)
+    assert_error(response, "no such window")
+
+
+def test_element_file_upload_state(session):
+    # 14.1 Step 3
+    session.url = inline("<input type=file value=Federer>")
+
+    element = session.find.css("input", all=False)
+    response = click(session, element)
+    assert_error(response, "invalid argument")
+
+
+def test_scroll_into_element_view(session):
```

Comment:
Can you please copy the tests that @whimboo wrote in https://searchfox.org/mozilla-central/source/testing/web-platform/tests/webdriver/tests/element_send_keys/scroll_into_view.py for this?

If you spot any gaps in test coverage, it would be good if you also filled those in for the Element Send Keys tests.

Reply:
The element_send_keys should be in another PR or you want to combine those tests into my PR as well?